### PR TITLE
GoLang.Go.1.19 version 1.19.3

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/92555)

### CODE REVIEW

**Comment by sinapis-amicus** [comment]
@microsoft-github-policy-service agree

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
For China — you can use `InstallerLocale: zh-CN`.

**Comment by sinapis-amicus** [comment]
> For China — you can use `InstallerLocale: zh-CN`.

Why?

**Comment by sinapis-amicus** [comment]
> For China — you can use `InstallerLocale: zh-CN`.

Done.

File: manifests/g/GoLang/Go/1/19/1.19.3/GoLang.Go.1.19.installer.yaml [hunk 1]
```diff
@@ -16,21 +16,24 @@ FileExtensions:
...
+- InstallerLocale: zh-CN
+  Architecture: x64
   InstallerType: wix
-  InstallerUrl: https://dl.google.com/go/go1.19.3.windows-amd64.msi
-  InstallerSha256: 056177B590426BDA3BDD7896B9DD36BD4B37211D16D46600DAB1FB430B1132F1
-  ProductCode: '{F358E20C-CDFE-4CE1-A8E7-F1B4A019922D}'
-- Architecture: x86
+  InstallerUrl: https://golang.google.cn/dl/go1.19.4.windows-amd64.msi
+  InstallerSha256: CFFAE9219EE1E8285AECF7038054BE25C6A6460C79813D5658BEE26640600233
+  ProductCode: '{6678FE64-D6D9-4791-9E05-74B23A42F6D0}'
```
Comment (stephengillie) [inline manifests/g/GoLang/Go/1/19/1.19.3/GoLang.Go.1.19.installer.yaml:24]:
```suggestion
  InstallerUrl: https://golang.google.cn/dl/go1.19.3.windows-amd64.msi
  InstallerSha256: 056177B590426BDA3BDD7896B9DD36BD4B37211D16D46600DAB1FB430B1132F1
  ProductCode: '{F358E20C-CDFE-4CE1-A8E7-F1B4A019922D}'
```

File: manifests/g/GoLang/Go/1/19/1.19.3/GoLang.Go.1.19.installer.yaml [hunk 2]
```diff
@@ -16,21 +16,24 @@ FileExtensions:
...
+- InstallerLocale: zh-CN
+  Architecture: x86
   InstallerType: wix
-  InstallerUrl: https://dl.google.com/go/go1.19.3.windows-386.msi
-  InstallerSha256: 8260FF43463470129804E3DD13C5728081FF617615913B69A10E8BCF77E5C709
-  ProductCode: '{338A6392-2631-43F0-AA6F-9DF933A297DA}'
-- Architecture: arm64
+  InstallerUrl: https://golang.google.cn/dl/go1.19.4.windows-386.msi
+  InstallerSha256: BEA9B95104A99A2D840A6AAE7CA7D20C0479684E38FE09EF00816F3808785A70
+  ProductCode: '{F32E3BD2-43A0-4DDE-B688-6AE8ABEE0033}'
```
Comment (stephengillie) [inline manifests/g/GoLang/Go/1/19/1.19.3/GoLang.Go.1.19.installer.yaml:30]:
```suggestion
  InstallerUrl: https://golang.google.cn/dl/go1.19.3.windows-386.msi
  InstallerSha256: 8260FF43463470129804E3DD13C5728081FF617615913B69A10E8BCF77E5C709
  ProductCode: '{338A6392-2631-43F0-AA6F-9DF933A297DA}'
```

File: manifests/g/GoLang/Go/1/19/1.19.3/GoLang.Go.1.19.installer.yaml [hunk 3]
```diff
@@ -16,21 +16,24 @@ FileExtensions:
...
+- InstallerLocale: zh-CN
+  Architecture: arm64
   InstallerType: wix
-  InstallerUrl: https://dl.google.com/go/go1.19.3.windows-arm64.msi
-  InstallerSha256: 0778A36C204EC20E0D7A0FEF359B2FF0B01CB9DD9D21E7FEFFC9F40A37F37F08
-  ProductCode: '{17729612-1B37-4F0B-9296-849AB1E8A41C}'
+  InstallerUrl: https://golang.google.cn/dl/go1.19.4.windows-arm64.msi
+  InstallerSha256: E5A72FE98F47E5CAA24230794482D0D8A23DDB9507AC904C2E704CB63A1E00E9
+  ProductCode: '{15777A18-783A-4AC9-9819-3169D5BBD28E}'
```
Comment (stephengillie) [inline manifests/g/GoLang/Go/1/19/1.19.3/GoLang.Go.1.19.installer.yaml:36]:
```suggestion
  InstallerUrl: https://golang.google.cn/dl/go1.19.3.windows-arm64.msi
  InstallerSha256: 0778A36C204EC20E0D7A0FEF359B2FF0B01CB9DD9D21E7FEFFC9F40A37F37F08
  ProductCode: '{17729612-1B37-4F0B-9296-849AB1E8A41C}'
```

**Comment by stephengillie** [comment]
Hi @sinapis

This PR both changes the package host from dl.google.com to golang.google.cn, and also changes the version from 1.19.3 to 1.19.4. These should be performed separately. Since this PR has 1.19.3 in the title, the new 1.19.4 version update should be reverted here and performed in another PR.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**